thrift-analytics-publisher/src/main/java/org/wso2/am/thrift/analytics/publisher/ThriftMetricEventBuilder.java

```diff
@@ -147,36 +147,17 @@ private void addCustomPropertiesAsAttributes(Map<String, Object> properties) {
             Object propertyValue = property.getValue();
 
             switch (propertyKey) {
-                case APPLICATION_CONSUMER_KEY:
-                case API_CONTEXT:
-                case API_RESOURCE_PATH:
-                case API_TIER:
-                case API_HOSTNAME:
-                case HOST_NAME:
-                case HOSTNAME:
-                case USER_TENANT_DOMAIN:
-                case THROTTLED_OUT:
-                case RESPONSE_TIME:
-                case SERVICE_TIME:
-                case BACKEND_TIME:
                 case RESPONSE_SIZE:
-                case PROTOCOL:
-                case SECURITY_LATENCY:
-                case THROTTLING_LATENCY:
-                case OTHER_LATENCY:
-                case LABEL:
-                case ERROR_CODE:
-                case ERROR_MESSAGE:
-                case SUBSCRIBER:
-                case THROTTLED_OUT_REASON:
-                case THROTTLED_OUT_TIMESTAMP:
-                case USERNAME:
```

Comment:
We had previously used fallback cases. These cases should **not** be removed. Rather, let's keep all of those, and just before the `default` case, let's put
```java
case RESPONSE_SIZE:
  eventMap.put(propertyKey,Long.parseLong(propertyValue.toString()));
```